index.html

```diff
@@ -4,7 +4,9 @@
     <meta charset="utf-8">
     <meta name="description" content="Labyrinth | Game ">
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+    <meta name="google-signin-client_id" content="<use your own API key and remove this rotated triangle brackets while filling api>">
```

Comment:
are you sure about the content of the meta tag?

Reply:
yes i cant deploy my official api KEY

Reply:
then how is the meta tag any help when it doesn't have the content? What are your views on removing it ?

Reply:
Nope I'll email you the key because here google is sending me warning if I deploy PR with API key

Reply:
since the web application (in this case, our game) is going to be public, and the client token only identifies the application as a unique identity (check references given below), it is **okay** to commit the api tokens.

It is a general caution to not commit api tokens to prevent abuse, but this is an identification token, so can be displayed.

***Check***
https://developers.google.com/identity/sign-in/web/sign-in
[How do client-side JS libraries for OAuth2 maintain secure authentication?
](https://stackoverflow.com/questions/24724238/how-do-client-side-js-libraries-for-oauth2-maintain-secure-authentication)
[application that uses OAuth and javascript](https://stackoverflow.com/questions/1221371/application-that-uses-oauth-and-javascript)